src/Helpers/SingleValue.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimPod\PhpSnmp\Helpers;
+
+use function array_shift;
+
+final class SingleValue
```

Comment:
I'm looking at the class and have no idea what it does. Can we fix namings?

Reply:
example pls ... I can't think of any

Reply:
describe what the class does. This looks like a VO while it is not

![image](https://user-images.githubusercontent.com/327717/83855751-d30b2c80-a718-11ea-92d1-3f7f577e888d.png)